src/SDKs/BatchAI/Management.BatchAI/Generated/SdkInfo_BatchAI.cs

```diff
@@ -1,22 +1,33 @@
 
-using System;
-using System.Collections.Generic;
-using System.Linq;
+// <auto-generated>
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
 
-internal static partial class SdkInfo
+namespace Microsoft.Azure.Management.BatchAI
 {
-    public static IEnumerable<Tuple<string, string, string>> ApiInfo_BatchAI
-    {
-        get
-        {
-            return new Tuple<string, string, string>[]
-            {
-                new Tuple<string, string, string>("BatchAI", "Clusters", "2018-03-01"),
-                new Tuple<string, string, string>("BatchAI", "FileServers", "2018-03-01"),
-                new Tuple<string, string, string>("BatchAI", "Jobs", "2018-03-01"),
-                new Tuple<string, string, string>("BatchAI", "Operations", "2018-03-01"),
-                new Tuple<string, string, string>("BatchAI", "Usage", "2018-03-01"),
-            }.AsEnumerable();
-        }
-    }
+  using System;
+  using System.Collections.Generic;
+  using System.Linq;
+
+  internal static partial class SdkInfo
+  {
+      public static IEnumerable<Tuple<string, string, string>> ApiInfo_BatchAI
+      {
+          get
+          {
+              return new Tuple<string, string, string>[]
+              {
+                new Tuple<string, string, string>("BatchAI", "Clusters", "2018-05-01"),
```

Comment:
Add the text `BatchAI_2018-05-01;` to the `AzureApiTag` node [here ](https://github.com/lliimsft/azure-sdk-for-net/blob/9c7155914cc93d38898227d6f3cb7bae5694ac15/src/SDKs/BatchAI/AzSdk.RP.props#L4)